templates/0.13.2/main/src/main.rs.gdpu

```diff
@@ -0,0 +1,41 @@
+use amethyst::{
+    core::transform::TransformBundle,
+    prelude::*,
+    renderer::{
+        plugins::{RenderFlat2D, RenderToWindow},
+        types::DefaultBackend,
+        RenderingBundle,
+    },
+    utils::application_root_dir,
+};
+
+struct MyState;
+
+impl SimpleState for MyState {
+    fn on_start(&mut self, _data: StateData<'_, GameData<'_, '_>>) {}
+}
+
+fn main() -> amethyst::Result<()> {
+    amethyst::start_logger(Default::default());
+
+    let app_root = application_root_dir()?;
+
+    let config_dir = app_root.join("config");
+    let display_config_path = config_dir.join("display.ron");
+
+    let game_data = GameDataBuilder::default()
+        .with_bundle(
+            RenderingBundle::<DefaultBackend>::new()
+                .with_plugin(
+                    RenderToWindow::from_config_path(display_config_path)
+                        .with_clear([0.34, 0.36, 0.52, 1.0]),
+                )
+                .with_plugin(RenderFlat2D::default()),
+        )?
+        .with_bundle(TransformBundle::new())?;
+
+    let mut game = Application::new(app_root.join("assets"), MyState, game_data)?;
```

Comment:
I like this change for the app_root.join("assets"), but can you create a folder (even if it's empty) just to show the location, like the current config directory? Just so it's crystal clear where it's meant to be. Thanks!

Reply:
git doesn't like empty folders unfortunately.

Reply:
ah, you can add an empty file called `.gitkeep` in the directory. It may appear in the final produced directory though, but that's not such a bad thing